persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConverationDAOImpl.kt

```diff
@@ -0,0 +1,89 @@
+package com.wire.kalium.persistence.dao
+
+import com.squareup.sqldelight.runtime.coroutines.asFlow
+import com.squareup.sqldelight.runtime.coroutines.mapToList
+import com.squareup.sqldelight.runtime.coroutines.mapToOneOrNull
+import com.wire.kalium.persistence.db.ConverationsQueries
+import com.wire.kalium.persistence.db.MembersQueries
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import com.wire.kalium.persistence.db.Conversation as SQLDelightConversation
+import com.wire.kalium.persistence.db.Member as SQLDelightMember
+
+class ConverationMapper {
+    fun toDAO(conversation: SQLDelightConversation): Conversation {
```

Comment:
Shouldn't this be something like `toModel` or `fromDatabaseRecord` or `fromDbEntry`, something like that? It doesn't actually map to a `DAO`.